libmscore/tempotext.h

```diff
@@ -46,11 +47,13 @@ class TempoText : public Text  {
 
       qreal tempo() const        { return _tempo;      }
       void setTempo(qreal v);
+      bool isRelative() { return _relative!=0; }
```

Comment:
I think worst case would be:
(from dotted whole to 64th)
(1.0/960) / (1.5/15.0) = 0.010416666666666666
probably enough far away from zero :)